chaudloader/src/mods/lua/lib/chaudloader/bnk.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{assets, mods, path};
+use mlua::ExternalError;
+
+pub fn new<'a>(
+    lua: &'a mlua::Lua,
+    mod_path: &std::path::Path,
+) -> Result<mlua::Value<'a>, mlua::Error> {
+    let table = lua.create_table()?;
+
+    table.set(
+        "load_bnk",
+        lua.create_function({
+            let mod_path = mod_path.to_path_buf();
+            move |_, (path,): (String,)| {
+                let path = path::ensure_safe(std::path::Path::new(&path))
+                .ok_or_else(|| anyhow::anyhow!("cannot read files outside of mod directory"))
+                .map_err(|e| e.into_lua_err())?;
+
+                let bnk_path = mod_path.join(&path);
+                if !bnk_path.exists() {
+                    return Err(anyhow::anyhow!("{} does not exist", bnk_path.display()).into_lua_err());
+                }
+
+                let base_filename = bnk_path.file_name().unwrap().to_str().unwrap();
+                const INVALID_BNK_NAMES: &[&str] = &["Init.bnk","Global.bnk","Vol1Global.bnk","Vol2Global.bnk","Voice1.bnk","DLC1.bnk","DLC2.bnk","EXE1.bnk","EXE2.bnk","EXE3.bnk","EXE4.bnk","EXE5.bnk","EXE6.bnk"];
```

Comment:
Would it make sense to reserve `chaudloader.bnk` also?

Reply:
If you wanted to future proof chaudloader generating bnk files.  I think the main issue with something like that at the moment is you would need the play event's ID to actually play the sound from a code mod and chaudloader does not currently have a good way to provide values it generates like that to a mod.

Reply:
That doesn't sound like it should be too difficult to find a solution for so I would suggest for now to reserve chaudloader.bnk also. We can always unreserve it later if it turns out to not be useful.